### ISSUE TITLE
Issue #3 - 번역 진행

### ISSUE THREAD

**Issue by nuxlear** [17-1-SKKU-OSS/178A#5]
- 여기는 번역 진행에 관련된 이슈를 적는 곳입니다. 

vim 에디터에서 92번째 줄을 기준으로 나누는 걸로 할게요

1 - 92 : 황준원
93 - 163 : 신재민

이렇게 담당해서 번역하죠

**Comment by nuxlear** [comment]
- 5/23 번역 초본 완성
- 5/30 리뷰 & pull request

향후 일정 계획입니다.

**Comment by whitecloudy** [comment]
확인했습니다. Whitecloud_mod 만들어서 번역 시작하겠습니다.

**Comment by whitecloudy** [comment]
일단 번역간에 고유명사들(Vundle, fish shelle등)은 음차표기 하지 않고 그냥 원문 그대로 둘께요.

**Comment by whitecloudy** [comment]
[ ]로 감싸진 공용 링크들은 우선 수정하지 않겠습니다. 추후에 공동으로 검수할때 일괄 수정하겠습니다.

**Comment by whitecloudy** [comment]
할당된 부분 번역 완료하였습니다. 확인 후 머징 부탁드립니다.

**Comment by nuxlear** [comment]
할당된 부분 저도 번역 완료했습니다. 확인 부탁드립니다.

**Comment by whitecloudy** [comment]
풀 리퀘스트 확인해서 머지 하시는건 팀징님 권한이니 팀장님이 하시면 될 것 같습니다.

**Comment by nuxlear** [comment]
네 그럼 머징하겠습니다

**Comment by whitecloudy** [comment]
1차 검수 완료하였습니다.

- 용어 통일
- 용어 정비
- 어색한 문장 정리
- 경어법 정리

**Comment by nuxlear** [comment]
넵 확인했습니다. 머지할게요

**Comment by nuxlear** [comment]
.vimrc 에 들어갈 내용 1차 번역 완료했습니다. 검수 부탁드릴게요

**Comment by whitecloudy** [comment]
.vimrc 내용 검수 완료했습니다.

**Comment by nuxlear** [comment]
머징했습니다

**Comment by nuxlear** [comment]
* 새로 번역하는 doc/vundle.txt 에 관해서 역할 분담입니다. 

1 - 195 & README_KR.md 링크 수정 : 황준원
196 - 411 : 신재민

**Comment by nuxlear** [comment]
doc/vundle_kr.txt 할당된 부분 1차 번역 완료했습니다. 확인 부탁드립니다

**Comment by whitecloudy** [comment]
네 확인했습니다. 지금 다른 일이랑 겹쳐서 번역에 시간을 내기가 쉽지가 않네요. 최대한 빠른 시일 내로 번역 완료해서 올리겠습니다.
